components/grid/export/events.md

```diff
@@ -246,7 +246,7 @@ The `OnAfterExport` event fires after the [OnBeforeExport](#onbeforeexport) even
 
 ### For Excel Export
 
-* `Stream` - `MemoryStream` - The output of the Excel export as a memory stream. 
+* `Stream` - `MemoryStream` - The output of the Excel export as a memory stream. The stream itself is finalized, so that the resource does not leak. It can be read from its available binary data.
```

Comment:
```suggestion
* `Stream` - `MemoryStream` - The output of the Excel export as a memory stream. The stream itself is finalized, so that the resource does not leak. The exported file can be read from its available binary data.
```

Reply:
Updated